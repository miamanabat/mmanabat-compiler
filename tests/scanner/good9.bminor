"?@?A!!!!! Ah!"
/* 7? no function 9! *
*//* okey dokey */
