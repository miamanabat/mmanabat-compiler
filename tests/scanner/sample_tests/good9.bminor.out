IDENTIFIER                              a
INTEGER_LITERAL                         5
STRING LITERAL           a >= 5
IDENTIFIER                              a
INTEGER_LITERAL                         5
STRING LITERAL            a > 5
IDENTIFIER                              a
INTEGER_LITERAL                         5
STRING LITERAL           a <= 5
IDENTIFIER                              a
INTEGER_LITERAL                         5
STRING LITERAL            a < 5
IDENTIFIER                              a
INTEGER_LITERAL                         5
STRING LITERAL           a == 5
IDENTIFIER                              a
INTEGER_LITERAL                         5
STRING LITERAL           a != 5
IDENTIFIER                              a
INTEGER_LITERAL                         5
IDENTIFIER                              a
IDENTIFIER                              b
STRING LITERAL        a || b is true
IDENTIFIER                              a
IDENTIFIER                              b
STRING LITERAL        a && b is true
